src/dealDashboard/deal-swap-status/deal-swap-status.ts

```diff
@@ -50,6 +50,7 @@ export class DealSwapStatus {
           header: "Congratulations!",
           message: "<p class='excitement'>You have successfully executed the token swap!</p>",
           confetti: true,
+          buttons: 1,
```

Comment:
For better readability, you can use here the `ShowButtonsEnum` from `AlertService`. Same as in line 35.

```suggestion
          buttons: ShowButtonsEnum.Primary,
```

Reply:
yeah, does look better